website/blog/2023-02-24-Deploy-databend-on-minio.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Deploying a single instance of Databend using MinIO
+description:  Using Databend to facilitate data analysis on MinIO
+slug:  deploying-databend-on-minio
+date: 2023-02-24
+tags: [beginner]
+category: Engineering
+cover_url: databend-minio-beginner-01.png
+image: databend-minio-beginner-01.png
+authors:
+- name: wubx
+  url: https://github.com/wubx
+  image_url: https://github.com/wubx.png
+---
+
+![Alt text](../static/img/blog/databend-minio-beginner-01.png)
+
+In this article, we will discuss how to deploy a single instance of Databend using MinIO for facilitating data analysis. MinIO is an object storage solution that is lightweight and easy to operate. Databend is a modern data warehouse designed for cloud architecture, built with Rust and open-source. It provides rapid elastic scaling and aims to create an on-demand, pay-as-you-go data cloud product experience.
+
+Open-Soruce Repo: https://github.com/datafuselabs/databend/
+
+Databend Docs: https://databend.rs
+
+## Databend Architecture
+
+![](../static/img/blog/databend-minio-beginner-01-1.png)
+
+Databend is architecturally divided into three layers: Meta Service Layer, Query Layer, and Storage Layer.
+
+- Meta Service Layer
+
+This layer stores permission definitions, table structure definitions, transaction management, table and data association, and the overall logic of data sharing. Cluster deployment is recommended in production.
+
+- Query layer
+
+This layer interacts directly with the user and the storage. The user interacts with Databend through SQL, and the Query Layer reads and writes the storage layer after receiving the user's request. This layer is not online real-time and can be pulled up when in use. It also supports dynamic expansion and contraction.
+
+- Storage Layer
+
+The Databend storage layer is the Databend Fuse Engine and is persisted using object storage in the cloud or self-built object storage. Databend uses the Parquet format on data block storage and implements min/max indexing, sparse indexing, bloom indexing, etc.
+
+
+### Databend support deployment environments
+
+| Environments| Databend |
+| ----------- | -------- |
+| AWS S3      | Yes     |
+| Google GCS  | Yes     |
+| Azure Blob  | Yes     |
+| Aliyun OSS  | Yes    |
+| Tencent COS | Yes     |
+| Huawei OBS  | Yes     |
+| MinIO       | Yes     |
+| Ceph        | Yes     |
+| Wasabi      | Yes     |
+| SeaweedFS   | Yes     |
+| QingStor    | Yes     |
+
+Detailed reference: https://databend.rs/doc/deploy/deploying-databend
+
+## Databend Single-machine deployment
+
+> In essence, the minio environment is relatively easy to set up. However, if you use in production, you can use the MinIO Cloud or AWS S3 environment to reduce hassle.
```

Comment:
Compared to the Databend deployment, a MinIO environment is easier to set up. In a production environment, MinIO Cloud or AWS S3 environment is recommended for a hassle-free experience.